crd/hack/update-codegen.sh

```diff
@@ -64,13 +79,7 @@ codegen_for () {
   output_pkg=${OUTPUT_BASE_PKG}/${1}
   apis_pkg=${APIS_BASE_PKG}/${1}/${2}
 
-  echo "Performing code generation for ${crd_name} CRD"
-  echo "Generating deepcopy functions and CRD artifacts"
-  go run sigs.k8s.io/controller-tools/cmd/controller-gen \
-          object:headerFile=${SCRIPT_ROOT}/hack/boilerplate.go.txt \
-          crd:crdVersions=v1 \
-          paths=${apis_pkg}/... \
-          output:crd:artifacts:config=${SCRIPT_ROOT}/config/crds
+  generate_config crd_name version
```

Comment:
The crd_name and version vars also don't seem to have values defined.

```suggestion
  generate_config $crd_name $version
```

Reply:
Done